crates/core/src/kvs/ds.rs

```diff
@@ -1,7 +1,7 @@
 use std::collections::BTreeMap;
 use std::fmt::{self, Display};
 #[cfg(storage)]
```

Comment:
We should either remove this or also feature-gate `new_with_clock`. It's using `Path` which is then breaking remote engine SDK tests as those engines don't depend on storage features.